styles/components/profile.css

```diff
@@ -0,0 +1,58 @@
+.profile {
+  margin: 0;
+  padding: 0;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  grid-template-rows: 1fr auto 1fr 1fr;
+}
+
+.profile__img {
+  margin: 0;
+  justify-self: start;
+  align-self: center;
+}
+
+.profile__img--boarder {
+  border-radius: 50%;
+}
+
+.profile__name {
+  padding-left: 10px;
+  align-self: center;
+}
+
+.profile__about {
+  text-align: justify;
```

Comment:
not sure if `text-align` works with justify. Do you want to center it?

Reply:
right, actually it is working 😊